### PR TITLE
Invoke previously defined command instead of running node directly

### LINKED ISSUE #10972: Issue with "Setup": Yarn 2.0 won't create the node_modules directory.
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Build_a_phone_with_peerjs/Setup

#### What information was incorrect, unhelpful, or incomplete?

If you are using Yarn 2.0, the node_modules directory is not created. 
If you add this line to your .yarnrc.yml file, the node_modules directory is created. 
nodeLinker: node-modules

Without making the change, when you issue 
node . 
as the writeup suggests, the program fails when it can't find the Express module. 

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/webrtc_api/build_a_phone_with_peerjs/setup`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Build_a_phone_with_peerjs/Setup
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/setup/index.md
* Last commit: https://github.com/mdn/content/commit/2279e5ae6c229c707a014a22aa1ec4635a0f981f
* Document last modified: 2021-09-14T18:08:09.000Z

</details>

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Summary

Fix #10972; supersedes #17642

#### Motivation

It's generally recommended to avoid running `node`, especially when using Yarn PnP since it simply doesn't work. For any slightly non-trivial task, including starting a server, abstraction with a custom command is recommended, and since we already have a command, let's use it instead.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/setup/index.md [hunk 1]
```diff
@@ -22,7 +22,6 @@ So let's get started by setting up the basis for our WebRTC-powered phone app.
       "name": "audio_app",
       "version": "1.0.0",
       "description": "An audio app using WebRTC",
-      "main": "server.js",
```
Comment (Josh-Cena) [inline files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/setup/index.md:25]: This is removed. It's slightly unconventional [citation needed; but I don't do it this way] to have a `main` for non-npm-package projects. Explicitly specifying which file to run is better IMO.

**Comment by Josh-Cena** [comment]
Also, the problem with having this kind of tutorial on MDN is that the web dev stack and practices renovate almost every six months, so it's super hard to keep the content up to date...

**Review by sideshowbarker — APPROVED** [review]
👍 Beautiful — thanks much

**Comment by sideshowbarker** [comment]
> Also, the problem with having this kind of tutorial on MDN is that the web dev stack and practices renovate almost every six months, so it's super hard to keep the content up to date...

Very true — and unfortunately that problem isn’t unique to this particular article; especially in the `Learn` subtree, we’ve got a number of guides which just inherently have a pretty short shelf life built in.

**Pull request merged**